frankip/book-share
# #161165927 Implement user registration

#### What does this PR do?
Implements user registration to the app
#### Description of Task to be completed?
As a user, I can register to the app using first name, last name, email address which should be unique and a password.
#### How should this be manually tested?
 - pull the branch `git fetch origin ft-user-creation-161165927`
 - checkout to the branch `git checkout ft-user-creation-161165927`
 - on postman make a `POST` request to `{{base-url}}/api/v1/registration` with data that resembles
the one below
```
{
	"first_name": "francis",
	"last_name": "kipchumba",
	"email": "frayn@gmail.com",
	"password": "12345qwerty"
}
```
It should give a success 201 status code or 409 error code in case of conflicting email
#### What are the relevant pivotal tracker stories?
[#161165927 ](https://www.pivotaltracker.com/story/show/161165927)
